.github/workflows/push.yml

```diff
@@ -143,6 +142,42 @@ jobs:
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.ACTION_MONITORING_SLACK }}
 
+  build-ipa:
+      runs-on: [self-hosted, macOS]
+      needs: paths-filter
+      if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/development' || needs.paths-filter.outputs.proxy == 'true' || needs.paths-filter.outputs.scenes == 'true' || needs.paths-filter.outputs.python == 'true'
+      steps:
+        - uses: actions/checkout@v3
+
+        - name: Build sampleGame ipa
+          run: |
+            # export ALTSERVER_PORT=13005
```

Comment:
If you don't need to export the `ALTSERVER_PORT` and `ALTSERVER_HOST` just remove this lines.